### ISSUE TITLE
Easier to navigate archive page & search?

### ISSUE THREAD

**Issue by jemostrom** [microdotblog/issues#105]
I've just imported two of my wordpress sites to micro.blog ... and suddenly realized that it can be kind tedious to find an old entry on that archive page. If I know approximate date it takes some scrolling to find it, here an archive page divided into year/month would help.

If I don't know the date ... well, then it becomes difficult. Some kind of search would be nice.

**Comment by manton** [comment]
@jemostrom I agree, the first step is going to be to break down that page so that it's easier to scroll to different years/months. Thanks! There is also a search box under Posts on Micro.blog on the web now if you just want to search your own posts.

**Comment by issimonwoods** [comment]
Maybe all themes should include a default Archive page with support for these improvements?
